src/classes/TDTM_Filter_TEST.cls

```diff
@@ -331,6 +331,11 @@ public with sharing class TDTM_Filter_TEST {
 	public static testmethod void lookupField() {
 		List<TDTM_Global_API.TdtmToken> tokens = setup();
         TDTM_Global_API.setTdtmConfig(tokens);
+
+        system.assertEquals(new List<TDTM_Global_API.TdtmToken>(), tokens);
+
+        //since we clear the token in setTdtmConfig now, we need to get the new tokens here
+        tokens = TDTM_Global_API.getTdtmConfig();
```

Comment:
Would it make sense that the `setup` method in this test class calls `TDTM_Global_API.getDefaultTdtmConfig()` instead of `TDTM_Global_API.getTdtmConfig()`? 

Also, do we really need this line? The tokens are explicitly set in line 353 below. Wouldn't that be enough?

Reply:
I think we can use TDTM_Global_API.getDefaultTdtmConfig() in the setup, should be ok. 

We need to have "tokens = TDTM_Global_API.getTdtmConfig()" here. The reason is that line 333 called setTdtmConfig(tokens). Since we cleared the token in setTdtmConfig, then the reference will be clear too. So, the tokens will be empty after line 333.  The line 353 will not work. 

I think we might need to mention in the documentation. The easiest way to update token is manipulate the tokens directly through getTdtmConfig. If user need to use setTdtmConfig, then they can't reuse the previous token reference anymore. They need to use getTdtmConfig to get a fresh set of token if they want to modify it.